dbld/rules

```diff
@@ -26,17 +26,17 @@ help:
 	@echo "  shell-<os>: start a shell in the specified OS"
 	@echo "  tarball: generate a tarball"
 	@echo ""
-	@echo "Supported OSs are: centos7, jessie, zesty."
+	@echo "Supported OSs are: centos7, jessie, zesty, xenial."
 	@echo ""
 	@echo "NOTE: this script requires that debian/rpm packaging are"
 	@echo "already merged to your tree, which is not there by default"
 	@echo "if you work in a git clone. These files live on the 'release'"
 	@echo "branch. Make sure you have that merged, OR work from a release tarball"
 
 
-deb: deb-zesty
+deb: deb-zesty deb-xenial
 
-shell: shell-zesty
+shell: shell-zesty shell-xenial
 
```

Comment:
I don't think these last two changes are needed, deb/shell are only meant for very simple invocation and should only use one of the targets and not all.

there's no point in executing two shells after one-another.